packages/ui/src/domainModules/collectionMammals/components/MammalManager/index.js

```diff
@@ -184,6 +198,14 @@ class MammalManager extends Component {
 
     if (Number.isInteger(parsedInteger)) {
       this.props.setCurrentTableRowNumber(parsedInteger)
+
+      const index = parsedInteger - 1
+      const specimenId = this.props.searchResult.items[index]
```

Comment:
to avoid error if there are no items in the search result it would be safest to do something like
```javascript
// this is one way to do it
const specimenId = this.props.searchResult && this.props.searchResult.items && this.props.searchResult.items[index]
// you can also use objectPath
const specimenId = objectPath.get(this.props, `searchResult.items.${index}`)

// after you get specimenId then only set it if it has a value
if (specimenId) {
  this.props.setFocusedSpecimenId(specimenId)
}
```

Reply:
ah, and it's a bug here. `items[index]` doesn't return the id, it returns an item. so we need to get the id from that item.